oauth2/oauth2-jose/src/test/java/org/springframework/security/oauth2/jwt/MappedJwtClaimSetConverterTests.java

```diff
@@ -43,19 +42,18 @@
  */
 public class MappedJwtClaimSetConverterTests {
 	@Test
-	public void convertWhenUsingCustomExpiresAtConverterThenIssuedAtConverterStillConsultsIt() {
-		Instant at = Instant.ofEpochMilli(1000000000000L);
-		Converter<Object, Instant> expiresAtConverter = mock(Converter.class);
-		when(expiresAtConverter.convert(any())).thenReturn(at);
-
-		MappedJwtClaimSetConverter converter = MappedJwtClaimSetConverter
-				.withDefaults(Collections.singletonMap(JwtClaimNames.EXP, expiresAtConverter));
+	public void convertWhenIssuedAtNullThenDerivedFromExpiresAt() {
+		MappedJwtClaimSetConverter converter =
+				MappedJwtClaimSetConverter.withDefaults(Collections.emptyMap());
 
+		Instant exp = Instant.now();
 		Map<String, Object> source = new HashMap<>();
+		source.put(JwtClaimNames.EXP, exp);
```

Comment:
@jzheaux The original test did not pass in any claims but the resulting claim set added `iss`. This does not seem right. If no claims are passed in than I would expect nothing to happen.

Reply:
Thanks for pointing this out. Did you mean the `iat` claim? This was to remain backward compatible with `NimbusJwtDecoderJwkSupport`, though there are perhaps other ways to achieve this.